include/dump_utils.hpp

```diff
@@ -64,9 +62,25 @@ inline void getValidDumpEntryForAttachment(
                                     "LogServices", "Dump", "Entries", entryID));
             return;
         }
+        dumpType = boost::algorithm::to_lower_copy(entryID.substr(0, pos));
         dumpId = entryID.substr(pos + 1);
     }
 
+    if (dumpType.empty())
+    {
+        // Other Types - continue to the next step with the empty type
+        std::string dumpEntryIdPath;
+        callback(std::move(dumpEntryIdPath), entryID, dumpType);
```

Comment:
if dump type is empty can we return error?

Reply:
I've verified the other cases.  They are considered as NotFound before.

OLD:
```
 % curl -k -X GET https://${bmc}/redfish/v1/Systems/INVALID/LogServices/Dump/Entries/INVALID/attachment
Not Found%
% curl -k -X GET https://${bmc}/redfish/v1/INVALID/INVALID/LogServices/Dump/Entries/INVALID/attachment
Not Found%
```

So, the change is better to return as an error.